--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-access-pointinterface.h	2008-09-30 07:35:52.000000000 +0200
+++ nm-access-pointinterface.h	2008-09-30 07:33:55.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-ACCESS-POINTINTERFACE_H_1222752952
-#define NM-ACCESS-POINTINTERFACE_H_1222752952
+#ifndef NM_ACCESS_POINTINTERFACE_H_1222729762
+#define NM_ACCESS_POINTINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-device-ethernetinterface.h	2008-09-30 07:35:53.000000000 +0200
+++ nm-device-ethernetinterface.h	2008-09-30 07:34:03.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICE-ETHERNETINTERFACE_H_1222752953
-#define NM-DEVICE-ETHERNETINTERFACE_H_1222752953
+#ifndef NM_DEVICE_ETHERNETINTERFACE_H_1222729762
+#define NM_DEVICE_ETHERNETINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-deviceinterface.h	2008-09-30 07:35:52.000000000 +0200
+++ nm-deviceinterface.h	2008-09-30 07:32:03.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICEINTERFACE_H_1222752952
-#define NM-DEVICEINTERFACE_H_1222752952
+#ifndef NM_DEVICEINTERFACE_H_1222729762
+#define NM_DEVICEINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
@@ -77,7 +77,7 @@
 
 public Q_SLOTS: // METHODS
 Q_SIGNALS: // SIGNALS
-    void StateChanged(uint new-state, uint old-state, uint reason);
+    void StateChanged(uint new_state, uint old_state, uint reason);
 };
 
 #endif
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-device-wifiinterface.h	2008-09-30 07:35:53.000000000 +0200
+++ nm-device-wifiinterface.h	2008-09-30 07:34:14.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICE-WIFIINTERFACE_H_1222752953
-#define NM-DEVICE-WIFIINTERFACE_H_1222752953
+#ifndef NM_DEVICE_WIFIINTERFACE_H_1222729762
+#define NM_DEVICE_WIFIINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-dhcp4-configinterface.h	2008-09-30 07:35:51.000000000 +0200
+++ nm-dhcp4-configinterface.h	2008-09-30 07:34:26.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DHCP4-CONFIGINTERFACE_H_1222752951
-#define NM-DHCP4-CONFIGINTERFACE_H_1222752951
+#ifndef NM_DHCP4_CONFIGINTERFACE_H_1222729762
+#define NM_DHCP4_CONFIGINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-ip4-configinterface.h	2008-09-30 07:35:52.000000000 +0200
+++ nm-ip4-configinterface.h	2008-09-30 07:28:43.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-IP4-CONFIGINTERFACE_H_1222752952
-#define NM-IP4-CONFIGINTERFACE_H_1222752952
+#ifndef NM_IP4_CONFIGINTERFACE_H_1222729762
+#define NM_IP4_CONFIGINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
@@ -20,6 +20,8 @@
 #include <QtCore/QVariant>
 #include <QtDBus/QtDBus>
 
+#include "generic-types.h"
+
 /*
  * Proxy class for interface org.freedesktop.NetworkManager.IP4Config
  */
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-manager-clientinterface.h	2008-09-30 07:35:52.000000000 +0200
+++ nm-manager-clientinterface.h	2008-09-30 07:34:43.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-MANAGER-CLIENTINTERFACE_H_1222752952
-#define NM-MANAGER-CLIENTINTERFACE_H_1222752952
+#ifndef NM_MANAGER_CLIENTINTERFACE_H_1222729762
+#define NM_MANAGER_CLIENTINTERFACE_H_1222729762
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
