--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-access-pointinterface.h	2008-06-07 15:29:30.000000000 +0200
+++ nm-access-pointinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-ACCESS-POINTINTERFACE_H_1212845370
-#define NM-ACCESS-POINTINTERFACE_H_1212845370
+#ifndef NMACCESSPOINTINTERFACE_H_1210621531
+#define NMACCESSPOINTINTERFACE_H_1210621531
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-device-802-11-wirelessinterface.h	2008-06-07 15:29:31.000000000 +0200
+++ nm-device-802-11-wirelessinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICE-802-11-WIRELESSINTERFACE_H_1212845371
-#define NM-DEVICE-802-11-WIRELESSINTERFACE_H_1212845371
+#ifndef NMDEVICE80211WIRELESSINTERFACE_H_1210621531
+#define NMDEVICE80211WIRELESSINTERFACE_H_1210621531
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-device-802-3-ethernetinterface.h	2008-06-07 15:29:30.000000000 +0200
+++ nm-device-802-3-ethernetinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICE-802-3-ETHERNETINTERFACE_H_1212845370
-#define NM-DEVICE-802-3-ETHERNETINTERFACE_H_1212845370
+#ifndef NMDEVICE8023ETHERNETINTERFACE_H_1210621532
+#define NMDEVICE8023ETHERNETINTERFACE_H_1210621532
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-deviceinterface.h	2008-06-07 15:29:30.000000000 +0200
+++ nm-deviceinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-DEVICEINTERFACE_H_1212845370
-#define NM-DEVICEINTERFACE_H_1212845370
+#ifndef NMDEVICEINTERFACE_H_1210621531
+#define NMDEVICEINTERFACE_H_1210621531
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-ip4-configinterface.h	2008-06-07 15:29:28.000000000 +0200
+++ nm-ip4-configinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-IP4-CONFIGINTERFACE_H_1212845368
-#define NM-IP4-CONFIGINTERFACE_H_1212845368
+#ifndef NMIP4CONFIGINTERFACE_H_1210621530
+#define NMIP4CONFIGINTERFACE_H_1210621530
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
@@ -20,6 +20,9 @@
 #include <QtCore/QVariant>
 #include <QtDBus/QtDBus>
 
+//HAND-EDIT
+#include "generic-types.h"
+
 /*
  * Proxy class for interface org.freedesktop.NetworkManager.IP4Config
  */
--- /space/kde/builds/trunk/KDE/kdebase/workspace/solid/networkmanager-0.7/dbus/nm-manager-clientinterface.h	2008-06-07 15:29:29.000000000 +0200
+++ nm-manager-clientinterface.h	2008-05-13 11:01:51.000000000 +0200
@@ -8,8 +8,8 @@
  * Do not edit! All changes made to it will be lost.
  */
 
-#ifndef NM-MANAGER-CLIENTINTERFACE_H_1212845369
-#define NM-MANAGER-CLIENTINTERFACE_H_1212845369
+#ifndef NMMANAGERCLIENTINTERFACE_H_1210621531
+#define NMMANAGERCLIENTINTERFACE_H_1210621531
 
 #include <QtCore/QObject>
 #include <QtCore/QByteArray>
